hw-7/task-4.js

```diff
@@ -0,0 +1,20 @@
+const spanValue = document.querySelector('#value');
+let counterValue = parseInt(spanValue.textContent, 10);
+
+const increment = () => {
+  // eslint-disable-next-line no-plusplus
+  counterValue++;
```

Comment:
Щоб лінтер не підсвічував помилку:
```suggestion
  counterValue += 1;
```